cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/mysql/MySQLBackendImpl.java

```diff
@@ -147,30 +151,131 @@ public void insertContextData(String dbName, String tableName, String fieldNames
         } catch (SQLException e) {
             throw new CygnusBadContextData(e.getMessage());
         } // try catch
+        
+        closeMySQLObjects(con, stmt);
     } // insertContextData
     
+    private CachedRowSet select(String dbName, String tableName, String selection)
+        throws CygnusRuntimeError, CygnusPersistenceError {
+        Statement stmt = null;
+        
+        // get a connection to the given database
+        Connection con = driver.getConnection(dbName);
+            
+        try {
+            stmt = con.createStatement();
+        } catch (SQLException e) {
+            throw new CygnusRuntimeError("SQLException, " + e.getMessage());
+        } // try catch
+        
+        try {
+            String query = "select " + selection + " from `" + tableName + "` order by recvTime";
```

Comment:
Maybe at this point a FIXME/TODO comment pointing to https://github.com/telefonicaid/fiware-cygnus/issues/1371 should be added